tools/cldr-apps/js/src/views/MainHeader.vue

```diff
@@ -153,6 +161,10 @@ export default {
         this.voteCountMenu = null;
         this.voteLevelChanged = 0;
       }
+
+      // only need CLA if logged in
+      this.needCla = !!user && !user.claSigned;
```

Comment:
I'm guessing that if DO_NOT_REQURE_CLA is true on the back end, then user.claSigned will get true (or truthy Date object?), even though that's sort of a fiction... OK, I've confirmed that

Reply:
yes